src/contractTest/java/uk/gov/hmcts/reform/wataskconfigurationapi/CcdCasePactTest.java

```diff
@@ -0,0 +1,108 @@
+package uk.gov.hmcts.reform.wataskconfigurationapi;
+
+import au.com.dius.pact.consumer.MockServer;
+import au.com.dius.pact.consumer.Pact;
+import au.com.dius.pact.consumer.dsl.PactDslJsonArray;
+import au.com.dius.pact.consumer.dsl.PactDslJsonBody;
+import au.com.dius.pact.consumer.dsl.PactDslWithProvider;
+import au.com.dius.pact.consumer.junit5.PactTestFor;
+import au.com.dius.pact.model.RequestResponsePact;
+import com.google.common.collect.Maps;
+import io.restassured.http.ContentType;
+import net.serenitybdd.rest.SerenityRest;
+import org.json.JSONException;
+import org.json.JSONObject;
+import org.junit.jupiter.api.Test;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.HttpStatus;
+
+import java.util.Map;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class CcdCasePactTest extends SpringBootContractBaseTest {
+
+    private static final String TEST_CASE_ID = "1607103938250138";
+    private static final String CCD_CASE_URL = "/cases/" + TEST_CASE_ID;
+
+    @Pact(provider = "ccdCase_Api", consumer = "wa_task_configuration_api")
```

Comment:
the provider has a weird naming convention: ccdCase_Api , does it have to be like that?